tests/data/rtc_gamma_golden.json

```diff
@@ -0,0 +1,95 @@
+{
+  "jobs": [
+    {
+      "name": "{{ name }}",
+      "job_parameters": {
+        "granule": "S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8",
+        "resolution": "10.0",
+        "radiometry": "sigma0",
+        "scale": "amplitude",
+        "speckle_filter": "true",
+        "dem_matching": "true",
+        "include_dem": "true",
+        "include_inc_map": "true"
+      },
+      "job_type": "RTC_GAMMA",
+      "description": "S1A_IW_20150621T120220_SVP_RTC10_G_saufed_"
```

Comment:
I assume this is a cheat to get around the mental gymnastics of mapping a S1 name to an RTC name (which has been a pain point for me since we adopted the new scheme).  I'd lean towards dropping `description`, especially if we're going to move forward with deprecating it.

Reply:
Yeah, I'm not actually using it right now, so I'll drop it.